build.gradle

```
@@ -172,8 +176,8 @@ jar {
     manifest {
         attributes(
                 "X-Kestra-Name": project.name,
-                "X-Kestra-Title": "Template",
-                "X-Kestra-Group": project.group + ".templates",
+                "X-Kestra-Title": "Youtube",
```

YouTube instead of Youtube here and everywhere to respect the official name please

```suggestion
                "X-Kestra-Title": "YouTube",
```